source/data/polls.js

```diff
@@ -28,7 +59,7 @@ const createPoll = async (request) => {
     return db.query('INSERT INTO public.polls (poll_title, is_closed, channel_name, channel_id, team_name, team_id, is_game, choices)' +
         'VALUES($1, $2, $3, $4, $5, $6, $7, $8)',
         [pollTitle, isClosed, channelName, channelId, teamName, teamId, isGame, {}]).then((pollResults) => {
-        return findOne({
+        return getByChannel({
```

Comment:
I still believe this extra DB call is unnecessary if you just return insertedPoll.id from the insert query using this query:
```
'INSERT INTO public.polls (poll_title, is_closed, channel_name, channel_id, team_name, team_id, is_game, choices)' +
        'VALUES($1, $2, $3, $4, $5, $6, $7, $8) returning id',
```

Reply:
Yes i haven't gotten to it yet, just parsing things out